omc3/run_kmod.py

```diff
@@ -285,7 +285,8 @@ def create_lsa_results_file(betastar_required, instruments_found, results_df, in
         exporting_columns=['LABEL', f'{BETA}{STAR}X', f'{ERR}{BETA}{STAR}X', f'{BETA}{STAR}Y', f'{ERR}{BETA}{STAR}Y']
         lsa_results_df=results_df[exporting_columns].rename(columns=dict(zip(exporting_columns, LSA_COLUMNS)))
     if instruments_found:
-        lsa_results_df = lsa_results_df.append(instrument_beta_df, sort=False, ignore_index=True)
+        lsa_results_df = pd.concat([lsa_results_df, instrument_beta_df],
```

Comment:
Same here, are these `TfsDataFrames`? If so, let's use `tfs.concat` unless we don't care about headers.